.github/workflows/rust.yml

```diff
@@ -10,6 +10,8 @@ env:
   CARGO_INCREMENTAL: 1
   # https://nexte.st/book/pre-built-binaries.html#using-nextest-in-github-actions
   CARGO_TERM_COLOR: always
+  RUST_MIN_STACK: 31457280
```

Comment:
Increasing the stack size for CI so the Keccak test of the prover does not fail.